plugins/csp-lod-bodies/shaders/brdfs/hapke1981.glsl

```diff
@@ -0,0 +1,52 @@
+// The Hapke BRDF to represent regolith, as defined in 10.1029/JB086iB04p03039 (DOI).
+
+// w: The average single-scattering albedo of a particle of the regolith,
+// i.e. the ratio of reflected energy to reflected plus absorbed energy at a particle, in range [0, 1].
+
+// h: The density parameter which controls the sharpness of the opposition peak.
+// A lower value describes a denser regolith, which leads to a sharper peak of the function.
+// "Sharpness" in this context is about how strong the opposition effect is at particular phase angles.
+// The range goes from 0 (excluded) to infinity (nonsense).
+
+// B_0: The amplitude of the opposition surge, i.e. simply how strong the effect is, in range [0, inf (nonsense)].
+
+// Some suggestions for some regoliths:
+// - Lunar regolith: w = 0.32, h = 0.4, B_0 = 1.8
+
+float $BRDF(vec3 N, vec3 L, vec3 V)
+{
+	float PI = 3.14159265358979323846;
+	
+    float g = acos(dot(L, V));
```

Comment:
You seem to have mixed tabs and spaces in the GLSL files. Could you please indent with two spaces consistently?